src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -67,7 +67,10 @@
       <div class="hmcts-primary-navigation__item">
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of (rightItems | async)">
-            <a class="hmcts-primary-navigation__link transparent-background" [ngClass]="item.ngClass" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
+            <!-- Find case link -->
+            <a class="hmcts-primary-navigation__link transparent-background" [ngClass]="item.ngClass" (click)="onEmitSubMenu(item)"
+              [attr.aria-current]="item.active ? true: null" [routerLink]="item.href" *ngIf="showFindCase && item.ngClass === 'hmcts-search-toggle__button'">{{item.text}}</a>
```

Comment:
@johnbenjamin-hmcts - this change is already in the `feature/global-search` branch; you just need to update your PR to point to it.